ciphers/brute_force_caesar_cipher.py

```diff
@@ -1,3 +1,6 @@
+LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Please import this from the `strings` module.

Reply:
Sure, I just moved the constants outside the function scope